.changeset/neat-schools-take.md

```diff
@@ -0,0 +1,12 @@
+---
+"@spectrum-css/clearbutton": major
+---
+
+#### S2 migration for clear button
+
+This migrates the clear button component to the latest Spectrum 2 designs. Custom properties have been remapped and added per the design specification.
+
+- The quiet variant has been remove as all clear buttons have a transparent background.
+- The over-background and static color variants have been removed.
+- Test and stories for deprecated variants have been removed.
+- The down state has been added, corresponding controls and tests have been added.
```

Comment:
What does that translate to for customers? Can you clarify in the docs here what the introduction of the down state looks like for customers adopting this change?